security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -41,6 +41,10 @@ const (
 	// The Istio secret annotation type
 	IstioSecretType = "istio.io/key-and-cert"
 
+	// Label which determines whether Citadel should watch namespace
+	NamespaceManagedLabel = "ca.istio.io/secret-controller-namespace"
```

Comment:
Labels and annotations should be in pkg/ something - since they may need to be used from multiple packages.
Possibly in api.

Also - please don't use 'secret-controller-namespace' - doesn't make sense from multiple reasons. 
SDS is not a secret controller, it may be remote (not in a namespace), etc.

Maybe ca.istio.io/address or ca.istio.io/provider  ( this MUST be discussed cross WG, since the pattern should apply to any other component, we can't use a different way for each WG. ).

Reply:
For `secret-controller`  Can you suggest better name and explains how it fits well with SDS? If there's some doc settled done, will be even better.

As for choosing namespace, we talked with @howardjohn and learned that existing configuration for installer is using namespace, the full url sounds flexible but does not exist yet.